src/renderer/features/settings/components/general/control-settings.tsx

```diff
@@ -36,6 +36,33 @@ export const ControlSettings = () => {
     const { setSettings } = useSettingsStoreActions();
 
     const controlOptions = [
+        {
+            control: (
+                <NumberInput
```

Comment:
Maybe make the number input smaller (and also add `Px`) or something to the right? See `Sample rate`

Reply:
Sure, I'll see about adding it there rather then in the language file.